src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -2022,5 +2023,35 @@ public String getVariableNamePattern() {
 					}
 				})
 				.serializer(new EnumSerializer<>(Attribute.class)));
+		EnumUtils<PlayerFishEvent.State> fishStateUtils = new EnumUtils<>(PlayerFishEvent.State.class, "fishing states");
+		Classes.registerClass(new ClassInfo<>(PlayerFishEvent.State.class, "fishingstate")
+				.user("fish(ing|) ?states?")
```

Comment:
(ing|) should be replaced with (ing)? 
Also, unrelated, but can you set the string on L1965 to `enchant(ment)? ?offers?`, someone mistook the string for a Skript pattern